var/spack/repos/builtin/packages/ferret/package.py

```diff
@@ -31,11 +31,10 @@ class Ferret(Package):
     """Ferret is an interactive computer visualization and analysis environment
        designed to meet the needs of oceanographers and meteorologists
        analyzing large and complex gridded data sets."""
-    homepage = "http://ferret.noaa.gov/Ferret/"
-    url      = "ftp://ftp.pmel.noaa.gov/ferret/pub/source/fer_source.tar.gz"
+    homepage = "http://ferret.pmel.noaa.gov/Ferret/home"
+    url      = "ftp://ftp.pmel.noaa.gov/ferret/pub/source/fer_source.v696.tar.gz"
 
-    version('6.96', '51722027c864369f41bab5751dfff8cc',
-            url="ftp://ftp.pmel.noaa.gov/ferret/pub/source/fer_source.tar.gz")
+    version('696', '51722027c864369f41bab5751dfff8cc')
```

Comment:
same here, from http://ferret.pmel.noaa.gov/Ferret/documentation you see developers saying 

> (Ferret v6.02)

so `6.96` was a good format.